openfl-tutorials/interactive_api/PyTorch_MedMNIST_2D/envoy/envoy_config_two.yaml

```diff
@@ -0,0 +1,11 @@
+params:
+  cuda_devices: []
+
+optional_plugin_components: {}
+
+shard_descriptor:
+  template: medmnist_shard_descriptor.MedMNISTShardDescriptor
+  params:
+    rank_worldsize: 2, 2
+    datapath: /home/nuc1/Documenti/DATASET
```

Comment:
Pointers to /home should be removed. Let's just keep one envoy config and delete this file.